src/diffusers/models/unet_2d.py

```diff
@@ -23,6 +23,38 @@ class UNet2DOutput(BaseOutput):
 
 
 class UNet2DModel(ModelMixin, ConfigMixin):
+    r"""
+    UNet2DModel is a 2D UNet model that takes in a noisy sample and a timestep and returns sample shaped output.
+
+    This model inherits from [`ModelMixin`]. Check the superclass documentation for the generic methods the library
```

Comment:
colud we add this to the other classes inheriting from `ModelMixin` as well?

Reply:
yup... just pushing now